src/pages/LoginPage.js

```diff
@@ -1,16 +1,102 @@
-import React from "react";
-import { Link } from "react-router-dom";
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import "../styles/LoginPage.css";
 
-function LoginPage() {
+export default function Login() {
+  const [formData, setFormData] = React.useState({ email: "", password: "" })
+  const [item, setItems] = React.useState([])
+  const [formErrors, setFormErrors] = React.useState({});
+  const history = useNavigate();
```

Comment:
as per the documentation The useNavigate hook returns a function that lets you navigate programmatically and not a browser history. rename this `const` to navigate. better for readability.

https://reactrouter.com/docs/en/v6/api#usenavigate

Reply:
Changed the naming convention.